.github/workflows/build-manylinux-container-images.yml

```diff
@@ -45,28 +45,32 @@ jobs:
         # Build containers for ppc64
         # - ARCH: ppc64
         YEAR:
-        - 2014
-        - _2_24  # PEP 600
-        - _2_28
+        - VALUE: 2014
+          TAG: 2025.01.19-2
+        - VALUE: _2_24  # PEP 600
+        - VALUE: _2_28
+          TAG: 2025.01.19-2
         include:
         - IMAGE:
             ARCH: x86_64
             QEMU_ARCH: amd64
-          YEAR: 1
+          YEAR:
+            VALUE: 1
         - IMAGE:
             ARCH: x86_64
             QEMU_ARCH: amd64
-          YEAR: 2010
+          YEAR:
+            VALUE: 2010
 
     env:
       LIBSSH_VERSION: 0.9.6
```

Comment:
Note to myself: we should eventually read this value from `pyproject.toml` or some other centralized location..